cdmtaskservice/models.py

```diff
@@ -180,19 +180,20 @@ def _check_fields(self) -> Self:
 
 
 class Parameters(BaseModel):
-    # TODO OPENAPI check f strings work with the open api docs
```

Comment:
They do not. I thought they worked in collections but I must've been thinking of examples or descriptions.